webapp/components/Hashtag/Hashtag.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <ds-tag>#{{ id }}</ds-tag>
+  <ds-tag>
+    <a v-bind:href="`/?hashtag=${id}`" target="_blank">#{{ id }}</a>
```

Comment:
```suggestion
    <nuxt-link :to="hashtagUrl">#{{ id }}</nuxt-link>
```
and implement a `hashtagUrl` computed property :wink: